test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,38 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsAreNotNull() throws Exception {
```

Comment:
If your test method name is `elementsAreNotNull`, it would be better to only contains all the non-null tests.

Reply:
oh, i see, thanks